lib/mongoid/touchable.rb

```diff
@@ -117,7 +147,7 @@ def define_relation_touch_method(name, association)
               if association.touch_field
                 # Note that this looks up touch_field at runtime, rather than
                 # at method definition time.
-                relation.touch association.touch_field
+                relation.touch(association.touch_field)
```

Comment:
I have a feeling this conditional is unnecessary... If the touch_field doesn't exist we'll just pass `nil` anyway which is the default parameter value... Might not be worth changing but just a note.

Reply:
It is unnecessary, great catch. I've removed it and added a comment instead.